apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.mustache

```diff
@@ -5,15 +5,19 @@ $(function() {
 
     new window.PLOrderBlocks(UUID, {
         maxIndent: {{max_indent}},
-        enableIndentation: {{check_indentation}}
+        inline: {{inline}},
+        enableIndentation: {{check_indentation}},
+        dropzoneLayout: "{{dropzone_layout}}",
+        blockLayout: "{{block_layout}}",
```

Comment:
I can't figure out where these are actually read inside the `PLOrderBlocks` class. If they aren't in fact needed, please remove them.

Reply:
Removed.